tools/libfrencutils/Makefile.am

```diff
@@ -9,6 +9,7 @@ libfrencutils_a_SOURCES = affinity.c \
                           constant.h \
                           create_xgrid.c \
                           create_xgrid.h \
+                          fms_platform.h \
```

Comment:
The git comparison shows fms_platform.h has been deleted.

Reply:
I just pushed an update removing this line from the makefile.